packages/ffe-symbols-react/src/index.d.ts

```diff
@@ -4,7 +4,6 @@ export interface SymbolProps {
     fill?: boolean;
     size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';
     weight?: 300 | 400 | 500 | 600;
-    color?: string;
     children: string;
     className?: string;
     ariaLabel: string;
```

Comment:
Ett inspill er att   ariaLabel også kan vare null. Alltså ikke undefined men null. Da har man tatt stilling. Dere kanskeje ønsker efterligna alt texten på bilder?

Reply:
ariaLabel føles mer naturlig før meg på Reat Component en en tom streng men det kanske bare er jag?

Reply:
Grunnen til at jeg satte det til tom string istedenfor null er at vi ønsker å tvinge brukerne til å sette aria-label, da det må være aria-hidden="true" eller aria-label for at man skal bestå WCAG lovkrav.  

Jeg tolket det som at hvis jeg har .isRequired i proptypes så tillater den ikke null, men det er kanskje feil?

Reply:
i typescript blir du tvinget precis som du ønsker. 
```
ariaLabel: string | null
```

Med proptypes ser det dessverre ut som man må lage noe hjemmelaget hvis man ønsker dette
![image](https://github.com/SpareBank1/designsystem/assets/2248579/af3d0b4a-753d-418a-b1a8-5f2cb4014527)

Reply:
Hmm..  Jeg er enig i at det er "ulogisk" å sende en tom string, men for meg gir det heller ikke så mye mening å sende ariaLabel={null}.

Frykten ved å gjøre det sånn at man ikke trenger å sende med ariaLabel i det hele tatt er at det blir masse ikoner der ute som burde vært lest opp for skjermleser, men som blir gjemt automatisk. Ved å tvinge folk til å ta en stilling til det, så må de hvertfall ta ett aktivt valg mens de jobber med komponenten, istedenfor at det går i glemmeboka.

Reply:
Lagde en typings funksjon som ser ut til å fungere som jeg vil? 
Man må sende inn ariaLabel, men aria-hidden settes til true dersom: `ariaLabel={null}` eller  `ariaLabel=""`

Reply:
null betyr ju bevist tom. undefined att den ikke fått en verdi... ungefær. Så med null har man tatt stilling